### PR TITLE
HAWKULAR-178 - Last used persona is the default upon login

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
For this to properly work, the following Accounts version needs to be used in Hawkular. As that version is already on Wildfly 10, it cannot be used.

This code, however, is prepared to fallback to the current behavior in such case, so, no updates to accounts are required for this to be merged.

**Comment by jpkrohling** [comment]
Accounts version that this feature will start to work as intended is: 2.0.7.Final-SRC-60d1a19656198451d44a429bc8646cdb48ad89da

**Comment by jpkrohling** [comment]
@mtho11 , could you please review ?

File: console/src/main/scripts/plugins/accounts/ts/persona.ts [hunk 1]
```diff
@@ -49,12 +49,42 @@ module HawkularAccounts {
 
     public loadData():void {
       this.loading = true;
-      this.currentPersona = this.HawkularAccount.Persona.get({id: 'current'},
+      let personaToGet = 'current';
+
+      let lastPersonaId = localStorage.getItem('lastPersona');
```
Comment (mtho11) [inline console/src/main/scripts/plugins/accounts/ts/persona.ts]: Aha localStorage. Good to know its being used. QE might also want to be aware of this.

**Comment by mtho11** [comment]
LGTM. @jpkrohling will you add the Accounts version to the pom for this PR as well?

File: console/src/main/scripts/plugins/accounts/ts/persona.ts (hunk 1 above)
Comment (jpkrohling) [inline console/src/main/scripts/plugins/accounts/ts/persona.ts]: Why? :)

**Comment by jpkrohling** [comment]
Not really, this can be merged without the update to Accounts, as it will just fallback to the default behavior. Once the WF10 task is finished, a newer version of Accounts will be used, so, this will work 'automatically'.

File: console/src/main/scripts/plugins/accounts/ts/persona.ts (hunk 1 above)
Comment (mtho11) [inline console/src/main/scripts/plugins/accounts/ts/persona.ts]: For setting up different default users. Perhaps not so interesting.

**Comment by mtho11** [comment]
Merged #697

**Pull request merged**